cranelift/codegen/src/isa/x64/mod.rs

```diff
@@ -447,36 +452,36 @@ mod test {
         //  0:   55                      push   rbp
         //  1:   48 89 e5                mov    rbp,rsp
         //  4:   83 ff 02                cmp    edi,0x2
-        //  7:   0f 83 27 00 00 00       jae    0x34
+        //  7:   0f 83 3b 00 00 00       jae    0x48
```

Comment:
This is due to the change in traversal order of the jump table: traversing the default label last means that it is emitted last.